Frontend/items.html

```diff
@@ -0,0 +1,202 @@
+            {item_id: 6, Name: 'potato', count: '6', purchased_on: '2016-01-23', detail:'purple and red potatoes'},
+            {item_id: 7, Name: 'chocolate', count: '2', purchased_on: '2016-01-19', detail:'dark chocolate, good for hot drink'}
+            ]
+
+        console.log(stock_array.length);
+
+
+    function createTable(){
+
+        var table_ref = document.getElementById('table_body');
+
+        for(var i = 0; i < stock_array.length; i++){
+            var new_row = table_ref.insertRow(0);
+            var new_cell_0 = new_row.insertCell(0);
+            var cell_id = document.createTextNode(stock_array[i].item_id);
+            new_cell_0.appendChild(cell_id);
+            var new_cell_1 = new_row.insertCell(1); 
+            var cell_name = document.createTextNode(stock_array[i].Name);
+            new_cell_1.appendChild(cell_name);
+            //alert('hello')
+            var new_cell_2 = new_row.insertCell(2);
+            var cell_count = document.createTextNode(stock_array[i].count);
+            new_cell_2.appendChild(cell_count);
+            var new_cell_3 = new_row.insertCell(3);
+            var cell_date = document.createTextNode(stock_array[i].purchased_on);
+            new_cell_3.appendChild(cell_date);
+            var new_cell_4 = new_row.insertCell(4);
+            var detail_button = document.createElement('button');
+             //add click handler to the detail button
+            detail_button.onclick = clickHandler(i);
+            var button_text = document.createTextNode('Click for details');
+            detail_button.appendChild(button_text);
+            new_cell_4.appendChild(detail_button);
+            var new_cell_5 = new_row.insertCell(5);
+            var update_button = document.createElement('button');
+            //add click handler to the update button
+            update_button.onclick = updateHandler(i);
+            var update_msg = document.createTextNode('Update');
+            update_button.appendChild(update_msg);
+            update_button.onclick = updateHandler(i);
+            new_cell_5.appendChild(update_button);
+            var new_cell_6 = new_row.insertCell(6);
+            var delete_button = document.createElement('button');
+            //add delete handler to the delete button
+            delete_button.onclick = deleteHandler(i);
+            var delete_msg = document.createTextNode('Delete');
+            delete_button.appendChild(delete_msg);
+            new_cell_6.appendChild(delete_button);
+
+            new_row.appendChild(new_cell_0);
+            new_row.appendChild(new_cell_1);
+            new_row.appendChild(new_cell_2);
+            new_row.appendChild(new_cell_3);
+            new_row.appendChild(new_cell_4);
+            new_row.appendChild(new_cell_5);
+            new_row.appendChild(new_cell_6);
+            table_ref.appendChild(new_row);
+
+        };
+    };
+
+    window.onload=createTable;
```

Comment:
are both this and the `<tbody id = 'table_body' onload = 'createTable();'></tbody>` needed? what happens if they have only one of them?